Bluejay.asm

```diff
@@ -1893,12 +1895,33 @@ scheduler_1024ms_switch_case_384ms:
 	sjmp scheduler_exit
 
 scheduler_1024ms_switch_case_512ms:
-	cjne A, #16, scheduler_exit
+	cjne A, #16, scheduler_1024ms_switch_case_640ms
 
 	; Stub for debug 2
 	mov Ext_Telemetry_L, #0CCh			; Set telemetry low value with temperature data
 	mov Ext_Telemetry_H, #0Ch			; Set telemetry high value on first repeated dshot coding partition
 
+	sjmp scheduler_exit
+
+scheduler_1024ms_switch_case_640ms:
+	cjne A, #20, scheduler_exit
+
+	mov Ext_Telemetry_L, #00h			; Set telemetry low value with temperature data
+
+	; Notify demag
+	jnb Flag_Demag_Notify, scheduler_1024ms_switch_case_640ms_demag_notified
+	orl	Ext_Telemetry_L, #01h
+	clr	Flag_Demag_Notify
+scheduler_1024ms_switch_case_640ms_demag_notified:
+
+	; Notify stall
+	jnb Flag_Stall_Notify, scheduler_1024ms_switch_case_640ms_stall_notified
+	orl	Ext_Telemetry_L, #02h
+	clr	Flag_Stall_Notify
+scheduler_1024ms_switch_case_640ms_stall_notified:
```

Comment:
Empty line should be before label, not after.

Reply:
Fixed